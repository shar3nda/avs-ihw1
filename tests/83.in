14
6 1 4 4 4 4 2 -1 4 0 10 6 2 -1