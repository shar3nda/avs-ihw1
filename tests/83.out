-1 4 0 10 6 2 -1